web/embed-dart.html

```diff
@@ -34,6 +34,11 @@
 
     <link rel="import" href="embed_components.html">
     <script type="application/dart" src="embed.dart"></script>
+    <style>
```

Comment:
Instead of this, let's define a `hidden` attribute on the `dartpad-dom-output` custom element.

In this page and in embed-inline.html, let's remove the `invisible` class and add the new `hidden` attribute.

In the definition of dartpad-dom-output, let's add a selector - something like:

```
[hidden] {
  display: none;
}
```

In order to hide the custom element when the hidden attribute is set.

You might also want to specify `:host` on that selector -